tests/redirects/map_globalconf.py

```diff
@@ -235,20 +235,27 @@
     url_test('/{m,{firefox/,}mobile}/faq/', 'https://support.mozilla.org/products/mobile'),
 
     # bug 885799, 952429
-    url_test('/projects/calendar/holidays.html', '/projects/calendar/holidays/'),
-    url_test('/en-US/projects/calendar/random/stuff/', '/projects/calendar/'),
-    # redirects don't catch real urls
-    url_test('/en-US/projects/calendar/', status_code=requests.codes.ok),
-    url_test('/en-US/projects/calendar/holidays/', status_code=requests.codes.ok),
+    url_test('/projects/calendar/holidays.html', 'https://www.thunderbird.net/en-US/calendar/holidays/', follow_redirects=True),
```

Comment:
These redirect tests would be a bit faster (and less prone to failure via external factors) if we removed `follow_redirects=True` e.g.:

```
url_test('/projects/calendar/holidays.html', 'https://www.thunderbird.net/calendar/holidays/'),
```

Reply:
Happy to make the change. Can you explain to me why this won't make the test fail?

Reply:
Notice he also changed the destination to remove the `en-US/` bit. Point is that we don't care what thunderbird.net does with the request, just that we're sending people to the proper beginning URL. So just test that bedrock is sending a 301 to the right URL and don't follow redirects. Not only is it slower to hit the destination, but it would also cause our tests to fail if that site were experiencing problems or changed their redirects.

Reply:
Ah, okay, the test does not require a positive response to pass, it just looks that the request went to the correct place?

Reply:
Right. These are redirect tests, so they just look at the response from bedrock to make sure it matches what we expect.